pull_request_template.md

```diff
@@ -1,23 +1,28 @@
-#### Feature / Issue / Improvement / Refactor (pick one)
+# Problem description
+Describe here what problem you are solving. Often this is (or should be) the same description is in the Linear ticket.
+Please don't just link to the Linear ticket here, don't force the reviewer to jump back and forth between GH and Linear.
```

Comment:
@danmarinescu description of this PR doesn't follow this rule

You might want to include this block in the description 
<img width="762" alt="Screen Shot 2022-09-22 at 9 10 39 AM" src="https://user-images.githubusercontent.com/452948/191755929-1bf88735-929c-4911-9412-7b80e8933ce3.png">

Reply:
I'll add the one missing sentence, the rest is captured in the other sections.

Reply:
I don't see it in this PR description

Reply:
Which sentence are you referring to specifically? I tried to capture all those points, in the process rephrasing some.

Reply:
> Please don't just link to the Linear ticket here, don't force the reviewer to jump back and forth between GH and Linear.

this PR description does include a link to the notion page, it's the same as the liner, right? this PR description should have the text from the Notion page, so you can see the issue without visiting the notion page (that might not exist later on)

Reply:
I see your point, very meta. I put the Notion link as a background reference for the reviewer. I don' think we need to worry about the Notion page not existing later on, as this change is easy to understand even without the context: this commit introduced a new PR template